### DISCUSSION TITLE
Render output file to a specified directory

### DISCUSSION THREAD

**Discussion by davidzgli** [quarto-dev/quarto-cli#8846] [Q&A] [answered]
### Description

I am trying to render a qmd file and save the output html file into a specific directory, not the current working directory.   My qmd file, render_test.qmd, is located in a subdirectory of a RStudio project, and I would like the output to be saved in the "render_out" directory.  

I tried to specify the --execute-dir argument. But the output is still saved in the current directory. 

```
quarto render render_test.qmd --to html --execute-dir render_out
```

The following command caused an error "ERROR: --output option cannot specify a relative or absolute path".

```bash
quarto render render_test.qmd --to html --output render_out/test.html
```

I know it is possible to move the html file manually after rendering.  But It is tedious and easy to make mistakes if many qmd files need to be rendered.  Thank you for any helps.

**Comment by mcanouil** [comment, marked as answer]
Duplicate of #2171.

Please see previous and more extensive discussion about this:

- https://github.com/quarto-dev/quarto-cli/discussions/2171

Especially my reply:

- https://github.com/quarto-dev/quarto-cli/discussions/2171#discussioncomment-4865286

**Comment by davidzgli** [comment]
@mcanouil Thank you for refer to the discussion #2171!   I went through the whole discussion but failed to find the solution to this issue.   I replied to your comments at #2171. Please take a look.
